go-controller/pkg/cni/types.go

```diff
@@ -77,4 +80,5 @@ type Server struct {
 	http.Server
 	requestFunc cniRequestFunc
 	rundir      string
+	KClient     kubernetes.Interface
 }
```

Comment:
L83: There is no need to export `KClient`, right? If so, it could just be `kclient`.

Reply:
Made private